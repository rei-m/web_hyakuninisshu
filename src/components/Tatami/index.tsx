import styled from 'styled-components';
import { withAppTheme } from '@src/styles';
import * as tatami from './tatami_part.png';

const Tatami = withAppTheme(styled.div)`
  background-image: url("${tatami}");
`;

export default Tatami;
