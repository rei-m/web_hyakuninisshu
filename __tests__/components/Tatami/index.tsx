import * as ReactTestRenderer from 'react-test-renderer';
import * as React from 'react';
import Tatami from '@src/components/Tatami';

describe('<Tatami />', () => {
  it('should render component', () => {
    const renderer = ReactTestRenderer.create(<Tatami>test</Tatami>);
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
