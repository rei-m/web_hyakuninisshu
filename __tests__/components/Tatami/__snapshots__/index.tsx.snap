// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tatami /> should render component 1`] = `
<div
  className="sc-bdVaJa jDcBRo"
>
  test
</div>
`;
